sharded_queue/drivers/fifo.lua

```diff
@@ -0,0 +1,146 @@
+local state = require('sharded_queue.state')
+local utils = require('sharded_queue.utils')
+local log = require('log') -- luacheck: ignore
+
+local method = {}
+
+local function tube_create(opts)
+    local space_opts = {}
+    local if_not_exists = opts.if_not_exists or false
+    space_opts.temporary = opts.temporary or false
+    space_opts.if_not_exists = if_not_exists
```

Comment:
I think we should not create `local if_not_exists` and use directly `opts.if_not_exists or false` to be consistent with another parameters

Reply:
copy-paste from original queue, thx

Reply:
Actually this is used in indices creation too, so let's leave it as is.